release_dashboard/lib/logic/cherrypick_state.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/proto.dart' as pb;
+
+const Map<pb.CherrypickState, String> cherrypickStates = <pb.CherrypickState, String>{
```

Comment:
This should be nested under cherryPickStateStr so it's no publically accessible (instead we should rely on the method)

Reply:
Oops, forgot to remove this.